include/grpc/event_engine/event_engine.h

```diff
@@ -491,7 +491,7 @@ void SetEventEngineFactory(
 /// using the previous factories.
 void EventEngineFactoryReset();
 /// Create an EventEngine using the default factory.
```

Comment:
I think this comment needs to be updated, since the factory is no longer going to be part of the public API.  It should probably just say something like "Creates a new EventEngine instance."